en/user/articles/130-forms/16-form-notifications.markdown

```diff
@@ -20,10 +20,9 @@ entry is submitted.
     name, or anything else informative to the recipient.
 
     **From Address:** The sender's email address. You can use something like 
-    `noreply@liferay-forms.com`, so that recipients don't try to reply. 
+    `noreply@example.com`, so that recipients don't try to reply. 
 
-    **To Address:** The recipient's email address (e.g., 
-    `test@liferay.com`). 
+    **To Address:** The recipient's email address (e.g., `test@example.com`).
```

Comment:
@rbohl - If you're in the middle of porting this doc, make sure to capture these changes to use `@example.com`. Thanks.

Reply:
I am, in fact, not currently porting, though I realise I should be :cold_sweat: